docs/workflow_solutions/shell.md

```diff
@@ -608,8 +608,56 @@ To change a directory and all of its contents recursively use `chgrp -hR <new-gr
 
 ### Manage researcher access to files and directories (`getfacl`, `setfacl`)
 
-<!-- markdownlint-disable MD046 -->
-!!! construction
+Access Control Lists (ACLs) provide the ability to set permissions for individual users or groups on specific files and directories.
 
-    Under construction.
-<!-- markdownlint-enable MD046 -->
+**Key ACl commands**:
+
+- `getfacl`: Command to get the ACL of a file or directory.
```

Comment:
Add a short sentence after this one saying something like "This lets you view what the ACLs are set to." The word "get" may not be obvious to some folks, and this alternative phrasing will help them build an understanding.